### PR TITLE
bump to websockets 7.4.6 to fix ReDoS vulnerability

### LINKED ISSUE #3196: Bump ws to patch out security vulnerability.
### Basic info:

- Node.js version -> any
- jsdom version -> any

### Minimal reproduction case

N/A

### How does similar code behave in browsers?

N/A

Currently ws 7.4.5 suffers from a ReDoS security vulnerability, should bump to latest fixed version to patch out vulnerability.

### PR DESCRIPTION
**Pull request by krystan**
move to websockets 7.4.6 to patch out ReDoS vulnerability. closes #3196

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -3650,7 +3650,12 @@ wrappy@1:
...
   integrity sha1-tSQ9jz7BqjXxNkYFvA0QNuMKtp8=
 
-ws@^7.4.5, ws@~7.4.2:
+ws@^7.4.6:
+  version "7.4.6"
+  resolved "https://registry.yarnpkg.com/ws/-/ws-7.4.6.tgz#5654ca8ecdeee47c33a9a4bf6d28e2be2980377c"
+  integrity sha512-YmhHDO4MzaDLB+M9ym/mDA5z0naX8j7SIlT8f8z+I0VtzsRbekxEutHSme7NPS2qE8StCYQNUnfWdXta/Yu85A==
+
+ws@~7.4.2:
   version "7.4.5"
```
Comment (TimothyGu) [inline yarn.lock:3659]: Please run `yarn upgrade`. As you can tell, 7.4.5 is still installed as dependencies of other packages.

Comment (krystan) [reply yarn.lock]: this has now been run

**Comment by ronmpatel** [comment]
@krystan can we merge this?

**Comment by krystan** [comment]
@ronmpatel I don't have the authority to merge it the workflow needs approval by someone

**Review by krystan — COMMENTED** [review]
this is done with things asked for done, needs workflow approval

**Review by TimothyGu — APPROVED** [review]

**Pull request merged**